tests/ui/macros/macro-metavar-expr-concat/raw-identifiers.rs

```diff
@@ -0,0 +1,16 @@
+#![feature(macro_metavar_expr_concat)]
+
+macro_rules! join {
+    ($lhs:ident, $rhs:ident) => {
+        let ${concat($lhs, $rhs)}: () = ();
+        //~^ ERROR `${concat(..)}` currently does not support raw identifiers
+        //~| ERROR `${concat(..)}` currently does not support raw identifiers
+        //~| ERROR `${concat(..)}` currently does not support raw identifiers
+    };
+}
+
+fn main() {
+    join!(r#_c, d);
+    join!(_e, r#f);
+    join!(r#_g, r#h);
```

Comment:
Could you add test cases for raw identifiers 1) passed through `tt` and 2) not passed through a macro parameter.